scripts/skills/perks/perk_rf_inspire.nut

```diff
@@ -0,0 +1,69 @@
+this.perk_rf_inspire <- this.inherit("scripts/skills/skill", {
+	m = {
+		// Config
+		MinDistanceAPRecovery = 1,
+		ActionPointsRecovered = 2,
+
+		// Temp
+		IsSpent = false,
+	},
+
+	function create()
+	{
+		this.m.ID = "perk.rf_inspire";
+		this.m.Name = ::Const.Strings.PerkName.RF_Inspire;
+		this.m.Description = ::Const.Strings.PerkDescription.RF_Inspire;
+		this.m.Icon = "ui/perks/rf_inspire.png";	// TODO
+		this.m.Type = ::Const.SkillType.Perk;
+		this.m.Order = ::Const.SkillOrder.Perk;
+		this.m.IsActive = false;
+		this.m.IsStacking = false;
+		this.m.IsHidden = false;
+	}
+
+	function isHidden()
```

Comment:
`function isHidden()` is unnecessary because this does not have the `StatusEffect` type.

Reply:
Ok I remove it. 
In the future we might want to add some marker to show whether I player already cashed in the triggered action point recovery this turn. But not now as I think its very unlikely that they forget that within a turn.